src/dice.class.php

```diff
@@ -156,10 +156,9 @@ function checkNewKeyNeeded() {
 	}
 
 	function createdice($numdice, $numsides) {
-		$i = 0;
-		while ($i < $numdice) {
-			$dice[$i] = mt_rand(1,$numsides);
-			$i++;
+		$dice = [];
+		for ($i = 0; $i < $numdice; $i++) {
+			$dice[$i] = mt_rand(1, $numsides);
```

Comment:
Could be something for a separate PR, but just throwing this out there...  Would you want to use a crypto RNG here to generate the dice rolls, since that's the bread-and-butter of a dice server?  I'm not sure if there's any kind of timing attack someone could make against a PRNG to get better dice rolls.

Reply:
Thought about this as well, but decided not to do so, because mt_rand seems to use a mersenne twister as it's random source which seems to be better than the standard rand functions. And should be sufficient (and faster) compared to cryptographic RNGs.
Also we use a mersenne twister for the local dice server as well...